### ISSUE TITLE
MEGAsync 4.6.0 doesn't detect file changes sometimes

### ISSUE THREAD

**Issue by ChrisFR06** [meganz/MEGAsync#639]
Hi,

Since I updated the MEGAsync client on my MacBook and on my Windows 11 tower at home, on both systems the client often doesn't detect changes or new files being created in my synced folder.

I have to manually click "force analysis" and wait for the client to restart and check if there is any new or updated files.

I never had this issue before (if I recall correctly, the version before 4.6.0 I had installed was 4.5.3), **this problem occurs only since I updated to 4.6.0.** I didn't know what to do to troubleshoot the issue so I tried clearing local cache (.debris), clearing SyncDebris, empyting the MEGA trash but nothing worked.

**TL;DR: MEGAsync doesn't detect new files/changes since 4.6.0, this seems to be the case on all operating systems (MacOS + Windows at least)**

**Comment by sakethv-mega** [comment]
Hello,

As far as we are aware, nothing to do with file notification detection changed with this version, for windows at least.  Could you please send us logs from the windows device:  
(1) reproduce the issue so that we can see any errors reported in the log 
(2) shut down the megasync app
(3) send us MEGAsync.log from the logs folder. Have a look in "C:\Users<you>\AppData\Local\Mega Limited\MEGAsync\logs\MEGAsync.log" 
And most importantly,
(4) tell us exactly what file/folder change you made that MEGAsync should have noticed. 

Also, please let us know if you have added any exclusions.

Thank you.
